lighthouse-core/scripts/git3po-rules/close-variability-issues.yaml

```diff
@@ -0,0 +1,26 @@
+repo: GoogleChrome/lighthouse
+pullRequests: false
+filters:
+  - type: issue
+    criteria:
+      state: open
+      labels:
+        $and:
+          - $includes: pending-close
+          - $includes: variability
```

Comment:
I feel like this is going to auto-trigger at some point for an issue that's about variability but shouldn't be deduped and then someone's going to add `pending-close` when the issue is wrapping up and it's going to be all confusing, but I guess that's unlikely to happen often :)

Reply:
heh, maybe we'll see :)